homeassistant/components/awair/const.py

```diff
@@ -0,0 +1,120 @@
+"""Constants for the Awair component."""
+
+from collections import namedtuple
+from datetime import timedelta
+import logging
+
+from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
+    CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+    CONCENTRATION_PARTS_PER_BILLION,
+    CONCENTRATION_PARTS_PER_MILLION,
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_ILLUMINANCE,
+    DEVICE_CLASS_TEMPERATURE,
+    TEMP_CELSIUS,
+    UNIT_PERCENTAGE,
+)
+
+AwairResult = namedtuple("AwairResult", ["device", "air_data"])
```

Comment:
We can use a dataclasses class instead of a namedtuple in Python 3.7+.
https://docs.python.org/3/library/dataclasses.html

Reply:
Have we officially dropped support for 3.6 yet? That's the reason I didn't use that in the first place.

Reply:
Yes, we're at Python 3.7+.

Reply:
That's really cool - I was going by what was in `tox.ini` for what language features I should use. I'll update to dataclasses :)